rotkehlchen/tests/unit/decoders/test_uniswapv3.py

```diff
@@ -1338,3 +1340,83 @@ def test_swap_on_polygon_pos(database, polygon_pos_inquirer, polygon_pos_account
         ),
     ]
     assert events == expected_events
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('optimism_accounts', [['0x9A539f692cDE873D6B882fc326c8d62D4cEA8048']])
+def test_add_liquidity_on_optimism(database, optimism_inquirer, optimism_accounts):
+    evmhash = deserialize_evm_tx_hash('0x96bd0e37e1734b5e73f9abdf30b39c4e4a6879667c2d01a7be2d95a85cc0b0cc')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
```

Comment:
is this fixing something that should be in the changelog or it was already there?

Reply:
there is already a changelog for this, I've updated it now